client/src/main/java/com/influxdb/client/internal/BucketsApiImpl.java

```diff
@@ -126,6 +128,18 @@ public Buckets findBuckets(@Nonnull final FindOptions findOptions) {
 
     }
 
+    @NotNull
```

Comment:
Use `javax.annotation.Nonnull` anotation
```suggestion
    @Nonnull
```